src/tendencies/vertical_diffusion_boundary_layer.jl

```diff
@@ -151,10 +50,10 @@ function surface_thermo_state(
     end
 end
 
-set_surface_thermo_state!(::Coupled, args...) = nothing
+set_surface_thermo_state!(::ManualUpdates, args...) = nothing
```

Comment:
Since we have added `PrescribedSurfaceField` and `update_surface_conditions!` for the coupler, all of the following functions should be deleted: `set_surface_thermo_state!`, `set_surface_inputs!`, and `get_surface_fluxes!`.